src/core/CMakeLists.txt

```diff
@@ -51,12 +82,12 @@ endif()
 add_library(kdstatemachineeditor_core SHARED ${LIB_SRCS})
 add_library(KDSME::Core ALIAS kdstatemachineeditor_core)
 target_link_libraries(
-    kdstatemachineeditor_core LINK_PRIVATE Qt${Qt_VERSION_MAJOR}::Quick # TODO: Get rid off this
-    LINK_PUBLIC Qt${Qt_VERSION_MAJOR}::Core Qt${Qt_VERSION_MAJOR}::Gui
+    kdstatemachineeditor_core LINK_PRIVATE Qt::Quick # TODO: Get rid off this
+    LINK_PUBLIC Qt::Core Qt::Gui
```

Comment:
For tests and examples it's fine but for the exported library itself, I don't recommend it.
https://bugreports.qt.io/browse/QTBUG-104112
https://bugreports.qt.io/browse/QTBUG-83774?focusedCommentId=665429&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-665429